lesson02/elb_asg/main.tf

```diff
@@ -0,0 +1,125 @@
+// Configure AWS Cloud provider
+provider "aws" {
+  region = "${var.aws_region}"
+}
+
+terraform {
+  backend "s3" {
+    bucket = "wizeline-academy-terraform"
+    key    = "terraform-academy/lessons2/terraform_lessons2.tfstate"
```

Comment:
let's add a placeholder for the `academy-user` so they don't forget to update it. (`academy-user-N/terraform-academy/lesson2/terraform_lesson2.tfstate`)

Reply:
ok good idea

Reply:
can we review this one?